oak-core/src/test/java/org/apache/jackrabbit/oak/query/AbstractQueryTest.java

```diff
@@ -312,10 +316,27 @@ protected void assertResultSize(String query, String language, long expected) {
 
     protected List<String> assertQuery(String sql, String language,
                                        List<String> expected, boolean skipSort) {
+        return assertQuery(sql, language, expected, skipSort, false);
+    }
+
+    protected List<String> assertQuery(String sql, String language,
+                                       List<String> expected, boolean skipSort, boolean checkSort) {
         List<String> paths = executeQuery(sql, language, true, skipSort);
-        assertResult(expected, paths);
+        if (checkSort) {
+            assertResult_Sorted(expected, paths);
+        } else {
+            assertResult(expected, paths);
+        }
         return paths;
+    }
 
+    protected static void assertResult_Sorted(@NotNull List<String> expected, @NotNull List<String> actual) {
```

Comment:
we should only use camelcase notation
```suggestion
    protected static void assertResultSorted(@NotNull List<String> expected, @NotNull List<String> actual) {
```